docs/docs/ecosystem/partake-in-governance.md

```diff
@@ -0,0 +1,74 @@
+# Democracy Usage Guide
+The following guide describes the different ways in which to use our governance system.
+
+
+## Democracy - How to propose
+The following guide describes the process of how to create a public governance proposal.
+
+### Requirements
+The following requirements have to be met:
+
+1. PICA available for transaction fees
+2. PICA available for voting
+3. PICA available for proposal submission
+4. PICA available for Pre- Image submission
+
+
+### Process
+By following the upcoming steps, you will create a governance proposal on Dali/Picasso/Composable.
+
+#### 1. Submitting a Pre- Image
+To submit a Pre-Image for a proposal, go to the democracy page of the PolkadotJS web interface. Then click on the "Submit preimage" button.
+Select the desired proposals extrinsic an outcome, note down the preimage hash, and press "Submit preimage".
+
+![](./partake-in-governance--images/1-governance--preimage-submission.png)
+
+
+#### 2. Submitting a proposal
+To submit a proposal, go to the democracy page of the PolkadotJS web interface. Then click on the "Submit proposal" button.
+Enter your "preimage hash" in the appearing text box, and select the amount of balance you want to lock with your proposal.
+
+
+#### 3. Getting proposal into referenda
+By now you will see your proposals in the "proposals" category on the democracy tab of the PolkadotJS web interface.
+Here, other people can endorse your proposal, and the proposal with most endorsements will get into referenda at the start of the next launch period.
+The launch period depends on the corresponding chain in use.
+
+| Chain | Launch Period |  
+| ----------- | ----------- |  
+| Dali | 1 Hour |  
+| Picasso | 5 Days |
+| Composable | 5 Days |
+
+To endorse a proposal, select the "Endorse" button next to a proposal on the PolkadotJS web interface.
+
+![](./partake-in-governance--images/3-governance--endorse.png)
+
+
+#### 4. Voting
+Now that your proposal is in referenda. People can vote on the outcome of the proposal.
+Voters can increase their voting power by locking their tokens up for up to a time of their wish, dependent on the corresponding chain in use.
+For the vote to be successful, it will need a `50% + 1` approval majority of all votes.
+
```

Comment:
Please expose the variable inside the democracy pallet that stores this value, in order to be easy for us to identify if we want to modify them.